.circleci/config.yml

```diff
@@ -53,25 +52,21 @@ workflows:
           filters:
             tags:
               ignore: /.*/
-          post-steps:
-              - jira/notify:
-                job_type: build
       - hmpps/helm_lint:
           name: helm_lint
       - hmpps/build_docker:
           context: [hmpps-common-vars]
           name: build_docker
+          jira_update: true
           filters:
             branches:
               only:
                 - main
                 - /.*deployable_branch.*/
-          post-steps:
-            - jira/notify:
-              job_type: build
       - hmpps/deploy_env:
           name: deploy_dev
           env: "dev"
+          jira_update: true
```

Comment:
will show in JIRA which environment the commit has been deployed to if the message is prefixed with the ticket number, like this commit